src/sentry/api/endpoints/user_notification_details.py

```diff
@@ -24,16 +24,28 @@ def get_attrs(self, item_list, user, *args, **kwargs):
         return results
 
     def serialize(self, obj, attrs, user, *args, **kwargs):
-        data = {option.key: option.value for option in attrs}
 
+        defaults = {
```

Comment:
this would probably be slightly better as a global defined at the top of the file.. like `USER_OPTION_DEFAULTS = ...` so it doesn't need to be redefined with each serialization